dubbo-spring-boot/dubbo-spring-boot-observability-starter/pom.xml

```diff
@@ -69,6 +69,11 @@
             <groupId>io.micrometer</groupId>
             <artifactId>micrometer-registry-prometheus</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.dubbo</groupId>
+            <artifactId>dubbo-metrics-prometheus</artifactId>
+            <version>${project.version}</version>
+        </dependency>
```

Comment:
Just add related dependency here directly. Do not transitively depend on the entire package.

Reply:
Here we use Prometheus as the default implementation, otherwise we need to introduce an additional dubbo-metrics-prometheus  package